modules_basic/markdown.js

```diff
@@ -1,26 +1,25 @@
-var markdown = require('ssb-markdown')
-var h = require('hyperscript')
-var ref = require('ssb-ref')
+const renderer = require('ssb-markdown')
+const fs = require('fs')
+const h = require('../h')
+const ref = require('ssb-ref')
 
 exports.needs = {
   blob_url: 'first',
   emoji_url: 'first'
 }
 
-exports.gives = 'markdown'
+exports.gives = {
+  markdown: true,
+  mcss: true
+}
 
 exports.create = function (api) {
-
-  function renderEmoji(emoji) {
-    var url = api.emoji_url(emoji)
-    if (!url) return ':' + emoji + ':'
-    return '<img src="' + encodeURI(url) + '"'
-      + ' alt=":' + escape(emoji) + ':"'
-      + ' title=":' + escape(emoji) + ':"'
-      + ' class="emoji">'
+  return {
+    markdown,
+    mcss: () => fs.readFileSync(__filename.replace(/js$/, 'mcss'), 'utf8')
```

Comment:
oh, it uses https://www.npmjs.com/package/static-eval and maybe that can handle your expression now?